### DISCUSSION TITLE
how to start remix with express ,is there  solution to sovle the devServer,building?

### DISCUSSION THREAD

**Discussion by 1371164370** [remix-run/remix#1115] [Q&A] [answered]
I'm start remix with express now,but the npm script `remix watch` depend on remix server,so there is no devServer wacthing when I run `start:dev`

**Comment by sergiodxa** [comment, marked as answer]
You need to run both commands, Remix watch will run the build process and start:dev will run the Expreaa server with NODE_ENV a development

**Reply by 1371164370** [reply]
thx,It does well ,  it would be better if I could run them in the same shell process，needn't to open two teminal
